aspnetcore/blazor/tutorials/movie-database-app/part-6.md

```diff
@@ -0,0 +1,177 @@
+
+The <xref:System.Data.Objects.DataClasses.EntityCollection%601.Contains%2A> method is run on the database, not in the C# code. The case sensitivity of the query depends on the database and the collation. For SQL Server, `Contains` maps to [SQL `LIKE`](/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. SQLite with default collation provides a mixture of case sensitive and case insensitive filtering, depending on the query. For information on making case insensitive SQLite queries, see the [Additional resources](#additional-resources) section of this article.
+
+Run the app and navigate to the movies `Index` page at `/movies`. The movies in the database load:
+
+![Mad Max movies before filtering in the movies Index page](~/blazor/tutorials/movie-database-app/part-6/_static/before-filtering.png)
+
+Append a query string to the URL in the address bar: `?titleSearch=road+warrior`. For example, the full URL appears as `https://localhost:7073/movies?titleSearch=road+warrior`, assuming the port number is `7073`. The filtered movie is displayed:
+
+!['The Road Warrior' Mad Max movie filtered using a query string in the browser's address bar](~/blazor/tutorials/movie-database-app/part-6/_static/query-string-filter-result.png)
+
+Next, give users a way to provide the `titleSearch` filter string via the component's UI. Add the following HTML under the H1 heading (`<h1>Index</h1>`). The following HTML reloads the page with the contents of the textbox as a query string value:
+
+```html
+<p>
+    <form action="/movies">
+        <input type="text" name="titleFilter" />
+        <input type="submit" value="Search" />
+    </form>
+</p>
+```
+
+:::zone pivot="vs"
+
+The next instruction is to run the app again. Save the file that you're working on. Visual Studio automatically recompiles and runs the app with your saved changes, so you can return to the existing browser window to see the result. You don't need to stop and restart the app for many types of code changes when using Visual Studio.
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+The next instruction is to run the app again. Close the browser window and in VS Code select **Run** > **Restart Debugging** or press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>F5</kbd> on the keyboard. VS Code recompiles and runs the app with your saved changes and spawns a new browser window for the app.
+
+:::zone-end
+
+:::zone pivot="cli"
+
+The next instruction is to run the app again. Stop the app in the command shell with <kbd>Ctrl</kbd>+<kbd>C</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>C</kbd> (macOS). Rerun the app with `dotnet watch`. Return to the existing browser window and refresh it to reload the app.
```

Comment:
Yes ... a blown update from a Find-'n-Replace 😈 when I was changing the command from `dotnet run`.

Reply:
```suggestion
The next instruction is to run the app again. Because the app is currently running with `dotnet watch`, saved changes are detected automatically and reflected in the existing browser window.
```